### DISCUSSION TITLE
How to deploy models to an Android application?

### DISCUSSION THREAD

**Discussion by hoanhvmetavi** [open-edge-platform/anomalib#1124] [Q&A] [answered]
First of all, thanks for the team's great work.

I wonder how can i deploy some model to run on an android application?
In current situation, the only way is export to onnx, then deploy onnx using onnxruntime.
Or are there any solution to achive this? Please let me know.

Best Regards!

**Comment by samet-akcay** [comment, marked as answer]
@hoanhvmetavi, you could potentially used the Arm plugin of OpenVINO for Arm-based Android devices. Here is an opencv blog explaining how you could do it.
https://opencv.org/blog/2021/09/09/face-recognition-on-android-using-openvino-toolkit-with-arm-plugin/

You could also refer to the OpenVINO documentation for more details.

I'm converting this to Q&A in Discussions. Feel free to continue the discussion there.
